setup.sh

```diff
@@ -33,19 +32,13 @@ git init
 git remote add origin https://github.com/hashbang/shell-etc.git
 git fetch --all
 git reset --hard origin/master
-git clean -f
-chmod 0600 /etc/sssd/sssd.conf
-
-# Disable users knowing about other users
-chmod o-r /var/run/utmp # Used by who(1)
```

Comment:
yeah. we should fix that :)
Could you remove commit 2552d79e8f1ad4cc88c99fda7f487b42347a0dcc from this PR for discussion elsewhere?